src/app/participant/forms.py

```diff
@@ -0,0 +1,6 @@
+__author__ = 'alex'
+from wtforms import Form, StringField, SelectField, TextAreaField
+
+
+class ParticipantsForm(Form):
+    name = StringField('Name')
```

Comment:
Oops! I shouldn't have committed this; I was going to work on forms but worked on the API instead. Forms aren't even really a WIP at this point; nothing has been started/done.